src/main/resources/kfs.ehcache.xml

```diff
@@ -0,0 +1,77 @@
+    <cache name="UniversityDate" maxEntriesLocalHeap="10000"/>
+    <cache name="SystemOptions" maxEntriesLocalHeap="10000"/>
+    <cache name="ObjectCode" maxEntriesLocalHeap="10000"/>
+    <cache name="ObjectType" maxEntriesLocalHeap="10000"/>
+    <cache name="BalanceType" maxEntriesLocalHeap="10000"/>
+    <cache name="HomeOrigination" maxEntriesLocalHeap="10000"/>
+    <cache name="AssetObjectCode" maxEntriesLocalHeap="10000"/>
+    <cache name="MileageRate" maxEntriesLocalHeap="10000" statistics="true"/>
+    <cache name="MenuLinks" maxEntriesLocalHeap="10000" timeToLiveSeconds="86760"/>
+
+    <cache name="KimTypeType" maxEntriesLocalHeap="5000" timeToLiveSeconds="1800"/>
+    <cache name="ResponsibilityType" maxEntriesLocalHeap="10000" timeToLiveSeconds="1800"/>
+    <cache name="ResponsibilityTemplate" maxEntriesLocalHeap="1000" timeToLiveSeconds="1800"/>
+    <cache name="GroupType" maxEntriesLocalHeap="25000" timeToLiveSeconds="1800"/>
+    <cache name="GroupHistoryType" maxEntriesLocalHeap="100000" timeToLiveSeconds="1800"/>
+    <cache name="GroupMemberType" maxEntriesLocalHeap="100000" timeToLiveSeconds="1800"/>
+    <cache name="EntityType" maxEntriesLocalHeap="100000" timeToLiveSeconds="1800"/>
+    <cache name="EntityNameType" maxEntriesLocalHeap="100000" timeToLiveSeconds="1800"/>
+    <cache name="PrincipalType" maxEntriesLocalHeap="100000" timeToLiveSeconds="1800"/>
+    <cache name="EntityPrivacyPreferencesType" maxEntriesLocalHeap="50000" timeToLiveSeconds="1800"/>
+    <cache name="CodedAttributeType" maxEntriesLocalHeap="100" timeToLiveSeconds="1800"/>
+    <cache name="PermissionType" maxEntriesLocalHeap="50000" timeToLiveSeconds="1800"/>
+    <cache name="PermissionTemplate" maxEntriesLocalHeap="10000" timeToLiveSeconds="1800"/>
+    <cache name="RoleType" maxEntriesLocalHeap="50000" timeToLiveSeconds="1800"/>
+    <cache name="RoleMembershipType" maxEntriesLocalHeap="200000" timeToLiveSeconds="1800"/>
+    <cache name="RoleMemberType" maxEntriesLocalHeap="200000" timeToLiveSeconds="1800"/>
+    <cache name="DelegateMemberType" maxEntriesLocalHeap="200000" timeToLiveSeconds="1800"/>
+    <cache name="RoleResponsibilityType" maxEntriesLocalHeap="200000" timeToLiveSeconds="1800"/>
+    <cache name="DelegateTypeType" maxEntriesLocalHeap="50000" timeToLiveSeconds="1800"/>
+    <cache name="KimAttributeType" maxEntriesLocalHeap="500" timeToLiveSeconds="1800"/>
+
+    <cache name="ParameterType" maxEntriesLocalHeap="2500" timeToLiveSeconds="3600"/>
+    <cache name="NamespaceType" maxEntriesLocalHeap="500" timeToLiveSeconds="3600"/>
+
+    <cache name="DocumentTypeType" maxEntriesLocalHeap="25000" timeToLiveSeconds="3600"/>
+    <cache name="RoutePathType" maxEntriesLocalHeap="25000" timeToLiveSeconds="3600"/>
+    <cache name="ExtensionDefinitionType" maxEntriesLocalHeap="5000" timeToLiveSeconds="3600"/>
+    
+    <!-- CU custom cache -->
+    <cache name="LockingIdCache" maxEntriesLocalHeap="10000"/>
```

Comment:
You may want to specify an explicit time-to-live value for this cache.

Reply:
OK, I will set timeToLiveSeconds="1800", 30 minutes seems reasonable for this particular cache